### ISSUE TITLE
Distinguish doc, xls and ppt

### ISSUE THREAD

**Issue by gillstrom** [gillstrom/is-doc#1]
Just noticed that this will be true for doc, xls and ppt. Is there any way to separate them?

**Comment by arthurvr** [comment]
Yeah, that's the reason I never published any of `is-doc`, `is-xls` or `is-ppt`, I never found this out. Looking at some generally pretty good [documentation](http://www.garykessler.net/library/file_sigs.html), there doesn't seem to be a way to find this out with its magic number.  

The magic number conflicts with `.wiz` ('wizards' used in their MS Office line IIRC) too, but I never even got a fixture of such file :p

**Comment by gillstrom** [comment]
I guess I'll have to remove this then :(

**Comment by arthurvr** [comment]
Dunno. Maybe other people know more than us :p
